docs/resources/Guild.md

```diff
@@ -889,11 +889,11 @@ Returns a list of [ban](#DOCS_RESOURCES_GUILD/ban-object) objects for the users
 
 ###### Query String Params
 
-| Field        | Type      | Description                                                                    | Default |
-| ------------ | -------   | ------------------------------------------------------------------------------ | ------- |
-| limit?       | number    | number of users to return (up to maximum 1000)                                 | 1000    |
-| before? *    | snowflake | consider only users before given user id                                       | null    |
-| after? *     | snowflake | consider only users after given user id                                        | null    |
+| Field        | Type      | Description                                                    |
+| ------------ | -------   | -------------------------------------------------------------- |
+| limit?       | number    | number of users to return (up to maximum 1000, the default).   |
+| before? *    | ?snowflake | consider only users before given user id                      |
+| after? *     | ?snowflake | consider only users after given user id                       |
```

Comment:
Can query strings be passed literally "null"?

Reply:
yes, tested